### ISSUE TITLE
[BUG] On auto disconnected no more photo displayed / Still assets to send but there are on server

### ISSUE THREAD

**Issue by alexandrewurtzfr** [immich-app/immich#1893]
#### I was disconnected from the app and when I reconnect to the app:

I no longer see any photos that were previously uploaded to the server.
(I still have a local copy. I have selected the right albums for the backup.)

There are assets left to send to the server but they have already been sent.
I see their thumbnails with crossed out cloud and top right counter at 108.

#### I took a new photo from the photo app and came back to immich:

The newly taken photo appears in the list of thumbnails. Still with 108 not sent (crossed out cloud and counter).

The new photo is sent to the server and some time later, all the photos that are on the server are reappeared.

In the list of thumbnails all photos have the valid cloud. The counter at the top right remains at 108. Same on the backup information page. In uploading file info there is the name of the missing file.

### The OS that Immich Server is running on

OS: Ubuntu 22.04.2 LTS / Docker: 23.0.1 (API: 1.42)

### Version of Immich Server

1.49.0 build.87

### Version of Immich Mobile App

1.49.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```
https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
```

### Your .env content

```
https://github.com/immich-app/immich/releases/latest/download/example.env
```

### Reproduction steps

```
- Get automatically disconnected from immich app (activate airplane mode and navigate in application)
- Reconnect (don't forget to deactivate airplane mode)
- Take photo with photo app
- Return to immich app
- Force kill immich app from recent app
- Reopen immich app
```

### Additional information

iOS 16.3 / iPhone 13

**Comment by alextran1502** [comment]
Hello, I cannot reproduce this scenario. Can you check the backup screen to make sure the albums are selected? The timeline only show the local assets from the selected album and the remote assets that were uploaded

**Comment by alexandrewurtzfr** [comment]
Yes the albums are selected.

The bug appears only by an automatic disconnection by app. And not by manual action when you press the button to disconnect.

However, when I login manually and log back in, I see all the photos.
There is construction of the timeline which is executed.

I noticed that the creation of the timeline does not do when there is an automatic disconnection.

**Comment by alextran1502** [comment]
This issue should be fixed in v1.52 release
